packages/main/src/plugin/tasks/progress-impl.ts

```diff
@@ -150,7 +155,11 @@ export class ProgressImpl {
     )
       .then(value => {
         // Middleware to capture the success of the task
-        t.status = 'success';
+        if (cancelRequest) {
```

Comment:
shouldn't we throw an error since it is cancelled :/

Reply:
I would tend to think it's not an error. If I cancel something, I expect at the end it's canceled, not an error.
Error = unexpected thing while here I really ask for that.